src/shared/components/query/studyList/StudyList.tsx

```diff
@@ -319,7 +324,19 @@ export default class StudyList extends QueryStoreComponent<IStudyListProps, {}>
                                 />
                             );
                         }
-    
+
+                        if (link.icon === 'tag') { // Metadata tag
+                            content = (
+                                <MetadataTooltip
+                                    key={i}
+                                    studyId={study.studyId}
+                                    mouseEnterDelay={0}
+                                    placement="top"
+                                    children={content}
```

Comment:
there are more conditions to the content above

Reply:
Did I misunderstand [the special meaning of `props.children` in JSX](https://reactjs.org/docs/jsx-in-depth.html#children-in-jsx) if I think that this would do the exact same thing or be even more robust?
```jsx
<MetadataTooltip
     key={i}
     studyId={study.studyId}
     mouseEnterDelay={0}
     placement="top"
>
    {content}
</MetadataTooltip>
```

Reply:
@fedde-s I've made the change, but it is a bit non-consistent now as just the element before (line 312) uses the same structure Riza initially wrote.

Reply:
To be more pragmatic and efficient, I suggest this should not to much further be discussed, especially the result is still the same and does not hurt the behaviour. 

If you want to refactor it, then refactor the rest of the code as well to keep the consistency and make new PR.

Reply:
Fully agree with @rnugraha, making that code more readable is out of scope here.

Reply:
Please do the same things in the same way to avoid messy code. This is a small effort that enhances readability, so I think it should be done.

Reply:
@pieterlukasse Done